src/components/Forms/Register.jsx

```diff
@@ -153,6 +153,8 @@ const Register = () => {
               user={user}
               setUser={setUser}
               text="Upload Resume"
+              sizeLimit="150 KB"
```

Comment:
make this an easier format so there's less confusing code when splitting it. u could try something like:

```
maxSize = {
    {
        K: 150
    }
}
``